composer/models/deeplabv3/README.md

```diff
@@ -0,0 +1,135 @@
+# DeepLabv3+
+[\[Example\]](#example) &middot; [\[Architecture\]](#architecture) &middot; [\[Training Hyperparameters\]](#training-hyperparameters) &middot; [\[Attribution\]](#attribution) &middot; [\[API Reference\]](#api-reference)
+
+[DeepLabv3+](https://arxiv.org/abs/1802.02611) is an architecture designed for semantic segmenation i.e. per-pixel classification. DeepLabv3+ takes in a feature map from a backbone architecture (e.g. ResNet-101), then outputs classifications for each pixel in the input image. Our implementation is a simple wrapper around [torchvision’s ResNet](https://pytorch.org/vision/stable/models.html#id10) for the backbone and [mmsegmentation’s DeepLabv3+](https://github.com/open-mmlab/mmsegmentation/tree/master/configs/deeplabv3plus) for the head.
+
+## Example
+
+```python
+from composer.models. import ComposerDeepLabV3
+
+model = ComposerDeepLabV3(num_classes=150,
+                          backbone_arch="resnet101",
+                          is_backbone_pretrained=True,
+                          backbone_url="https://download.pytorch.org/models/resnet101-cd907fc2.pth",
+                          sync_bn=False
+)
+```
+
+## Architecture
+
+Based on [Encoder-Decoder with Atrous Separable Convolution for Semantic Image Segmentation](https://arxiv.org/abs/1802.02611)
+
+<div align=center>
+<img src="https://storage.googleapis.com/docs.mosaicml.com/images/models/deeplabv3_v2.png" alt="deeplabv3plus" width="650">
+</div>
+
+
+- **Backbone network**: converts the input image into a feature map.
+    * Usually ResNet-101 with the strided convolutions removed in stage 3 and 4.
```

Comment:
```suggestion
    * Usually ResNet-101 with the strided convolutions converted to dilations convolutions in stage 3 and 4.
```

previous wording suggested the conv layers were actually removed.

Reply:
Sounds good!